flit/upload.py

```diff
@@ -85,7 +85,9 @@ def get_repository(name=None, cfg_file="~/.pypirc"):
     3. keyring
     4. Terminal prompt (store to keyring if available)
     """
-    repos_cfg = get_repositories(cfg_file)
+    log.info(pypirc_path)
+    repos_cfg = get_repositories(pypirc_path)
+    log.info(repos_cfg)
```

Comment:
```suggestion
    log.debug("Loading repositories config from %r", pypirc_path)
    repos_cfg = get_repositories(pypirc_path)
```

Log messages at info level and above are visible to the user by default, so I like to be fairly minimal with them. Debug level is only shown if you add `--debug`, which seems about right for this.

`repos_cfg` may contain passwords, so it's probably best that we don't log that at all! You don't want it to give away your password in a CI log.

Reply:
Thanks, good catch. You are completely right; this was left over from my debugging, I'll change that.